components/SlideEach/SlideEach.tsx

```diff
@@ -0,0 +1,21 @@
+import { SlideIn } from "components/SlideIn";
+import { ISlideIn } from "components/SlideIn/SlideIn";
+import React, { FC, ReactNode } from "react";
+
+interface ISlideEach extends Omit<ISlideIn, "children"> {
```

Comment:
Why did you use Omit here? Omit is unnecessary. There are no type clashes. 
Also, children prop will get inherted from ISlideIn. its also unecessary

correct these 2 unneccasry things

Reply:
resolved